packages/ui/src/components/Checkbox/consts.ts

```diff
@@ -0,0 +1,5 @@
+export enum CheckboxSize {
+  MINI = 'mini', // Size: M, 12 x 12
+  SMALL = 'small', // Size: L, 16 x 16
```

Comment:
That's not strict, but it would be better to name enum's variables in PascalCase. And maybe name enum itself with `S` prefix, like a component.
Also I think it would be better to move comments like 'Size: M 12 x 12' to doc comments so they will be visible later in VSCode:

```ts
export enum SCheckboxSize {
  /**
   * Size: M, 12 x 12
   */
  Mini = 'mini',
  // ...
}
```